tox.ini

```diff
@@ -1,28 +1,28 @@
 [tox]
+skipsdist = True
+usedevelop = True
 envlist =
-  py{34,35,36}-django111-wagtail23
-  py{35,36}-django21-wagtail23
-  py{35,36,37}-django{21,22}-wagtail27
+    py{35,36}-dj111-wt23
```

Comment:
Arguably we can drop 2.3 [as of this month](https://github.com/wagtail/wagtail/wiki/Release-schedule), but no harm in keeping it out of friendliness as long as tests are still passing.